help/de/Help_ch_Account-Actions.xml

```diff
@@ -1013,22 +1038,23 @@
       zuvor beschriebenen Dialog <xref linkend="acct-create"></xref>. Um das Dialogfenster
       <guilabel>Konto bearbeiten</guilabel> zu öffnen, wählen Sie
       <menuchoice>
-        <guimenu>Bearbeiten</guimenu> <guimenuitem>Konto bearbeiten</guimenuitem>
+        <shortcut>
+        <keycombo>
+          <keycap>Strg</keycap><keycap>E</keycap>
+        </keycombo>
+        </shortcut> <guimenu><accel>B</accel>earbeiten</guimenu> <guimenuitem>Konto
+        <accel>b</accel>earbeiten</guimenuitem>
       </menuchoice>
-      (
-      <keycombo>
-        <keycap>Strg</keycap><keycap>E</keycap>
-      </keycombo>
-      ) oder klicken Sie die <guibutton>Bearbeiten</guibutton> Schaltfläche in der
+      oder klicken Sie die <guibutton>Bearbeiten</guibutton> Schaltfläche in der
       <emphasis>Werkzeugleiste</emphasis> des Fensters <quote>Account</quote> an.
     </para>
   </sect1>
 
   <sect1 id="chart-renumber">
     <title>Neunummerierung der Unterkonten</title>
 
-    <para>Verwenden Sie <guilabel>Unterkonten neu nummerieren...</guilabel>, um die Kontonummern für alle
-      Unterkonten des aktuellen Kontos neu zu schreiben. Sie haben zum Beispiel die folgende
+    <para>Verwenden Sie <guilabel>Unterkonten neu nummerieren&hellip;</guilabel>, um die Kontonummern für
```

Comment:
Da wir utf-8 verwenden braucht die Ellipse nicht mehr gequotet werden. Du kannst einfach [AltGr]+{.] tippen.

Reply:
@fellen: ich stehe auf dem Schlauch. Mit [AltGr]+[.] werden drei Punkte eingefügt. Was hat das mit &hellip; und der Beschriftung von Menüeinträgen zu tun?

Reply:
Ersetze `&hellip;`durch `…`.
Das Ergebnis ist dasselbe. und  geht doch viel schneller

Aufpassen muß man nur wenn `<` oder `&` im Text erscheinen sollen, weil die in Dokbook Tags und Entitäten einleiten. Alles andere kann und sollte wegen der Lesbarkeit direkt eingegeben werden.

Als die Dateien noch in  ISO-8859-1 kodiert waren, ging das noch nicht.Daher gibt es noch viele `&hellip;` etc. im Text.

Reply:
ok. verstanden und korrigiert.